src/ServiceControl.UnitTests/ApprovalFiles/APIApprovals.RootPathValue.approved.txt

```diff
@@ -1,5 +1,5 @@
 {
-  "description": "testDescription",
+  "description": "The management backend for the Particular Service Platform",
```

Comment:
I don't think these changes are necessary. The test sets explicit fake values.

Reply:
Turned out that some test muck around with the config file, did a clean rebuild and only ran the approvals so they should be back to the original state now